client/app/components/composites/ListingCard/ListingCard.js

```diff
@@ -0,0 +1,144 @@
+import { Component, PropTypes } from 'react';
+import r, { a, div, img } from 'r-dom';
+import classNames from 'classnames';
+import { tint } from '../../../utils/colors';
+
+import Avatar from '../../elements/Avatar/Avatar';
+import css from './ListingCard.css';
+import noImageIcon from './images/noImageIcon.svg';
+import distanceIcon from './images/distanceIcon.svg';
+
+const MINIMUM_DISTANCE = 0.1;
+const PRECISION = 2;
+const TINT_PERCENTAGE = 20;
+
+const sigFigs = function sigFigs(n, sig) {
+  return parseFloat(n.toPrecision(sig));
+};
```

Comment:
We should probably use numeral.js or something for formatting, to get thousands separators etc correct. But we can do that also later.